### ISSUE TITLE
Inclusion rules for seas and related water bodies

### ISSUE THREAD

**Issue by axelboc** [axelboc/anki-ultimate-geography#346]
As agreed in https://github.com/axelboc/anki-ultimate-geography/issues/137#issuecomment-620764010, and since discussion has started again on this topic in #137, I'm opening this issue to discuss inclusion rules for water bodies (oceans, seas, gulfs, straits, etc.) I'm leaving lakes and rivers out for now (see why down below).

Like for political entities (#306 + #312), we need to identify which types of water bodies to include, and then, for each type of entities:

1. find a source article on Wikipedia that lists them as exhaustively as possible;
2. come up with precise, practical inclusion rules to decide which ones and how many to include (with maps only this time -- no capitals and flags to worry about... 😄);
3. document the source article and inclusion rules in `CONTRIBUTING.md`.

#### Oceans

Like for continents, this one's easy: https://en.wikipedia.org/wiki/Ocean#Oceanic_divisions. Since we're not a historical deck, there's no debate that the world has five oceans and all five of them are to be included ... which they already are! They even have brand new maps #325! ✨ 

#### Marginal seas

Wikipedia's [list of seas](https://en.wikipedia.org/wiki/List_of_seas) seems like a good reference. The problem is that it lists gulfs, bays, straits, channels, etc. within the same _Marginal seas_ heading, without further categorisation. Apparently the terms sea, gulf, bay, sound, etc. are used inconsistently (as you've rightly pointed out in https://github.com/axelboc/anki-ultimate-geography/issues/137#issuecomment-650528659, @aplaice), so I don't think we'll be able to list them independently and apply different criteria to them.

The good news is that the list already excludes lakes with "Seas" in their names, like the Sea of Galilee or the Aral Sea. Also, we might be able to separate out straits, channels, passages, and so on. If that's indeed the case, then I think using the surface area as the sole inclusion criteria for the remaining seas, gulfs, etc. would work pretty well.

#### Straits, channels, passages, etc.

Same list as above. As @aplaice mentioned, straits between continents would be the easy pick. Also, since I can only see two channels, the English Channel and the [Mozambique Channel](https://en.wikipedia.org/wiki/Mozambique_Channel), I reckon we can include them as well. The remaining bodies are more debatable.

#### Lakes

I'd be inclined to leave these alone for now, for the following reasons:

- The [list of lakes](https://en.wikipedia.org/wiki/List_of_lakes) is quite massive and "in the process of an expansion or major restructuring".
- Sourcing the surface area of all of these is going to be quite challenging...
- Inclusion criteria are not obvious if some people are keen on keeping the Sea of Galilee and the Dead Sea.

#### Rivers, deltas, fjords, etc.

Since we don't have any in the deck yet, let's leave them out of the scope of this issue.

**Comment by Sir-Casm** [comment]
Sort of modified repost of https://github.com/axelboc/anki-ultimate-geography/issues/137#issuecomment-632699020 as I didn't get any feedback there:
Regarding the bodies of water, I am currently using the ["Rivers, Lakes, Seas, and Oceans"](https://ankiweb.net/shared/info/1399758390)-Deck mentioned https://github.com/axelboc/anki-ultimate-geography/issues/137#issuecomment-533351257. I deleted the cards that were already present in the UG deck, translated the rest into German and also modified it to match the UG cards style (although I also changed the UG design a bit to e.g. get the corresponding Wiki articles embedded in an iFrame). Most of the data in the deck is on your "ignored for now" list (e.g. biggest lakes and longest rivers), though. 
The mentioned "Rivers, Lakes, Seas, and Oceans"-Deck has really good maps of the water bodies. The deck itself is an export of the ["Rivers, Lakes and Seas"](https://www.memrise.com/course/28784/rivers-lakes-and-seas/) deck by azrael42 on Memrise.
Most of the location maps were created by azrael42 and are based on the work of the following Wikipedia Commons users:
Africa: Sting (Eric Gaba), Middle East, Central America and Australia: Виктор В, Europe: Alexrk2, New Zealand, Sulawesi: NordNordWest, Japan: Chumwa, all remaining maps (!): de.wikipedia.org/wiki/Benutzer:Uwe_Dedering
Please see here for [more information](https://www.memrise.com/course/28784/rivers-lakes-and-seas/20/) on the original deck." (from the description of the anki deck) on the original deck." (from the description of the anki deck).
Maybe there is a chance to "just include" this deck into UG, with the appropriate mentionings (I have no clue about the license stuff)? If this is the case and you, at some point in time, plan to include rivers and lakes into the UG deck, I can provide you with my translated and slightly modified privately used Deck (.apkg), to not do the translation works twice (at least into German). I can extract a double language deck (with both English and German names included).
Happy to get some feedback on that.

**Comment by axelboc** [comment]
Thanks @Sir-Casm, we'll keep an eye on the "Rivers, Lakes and Seas" deck. Some of the maps look like they match ours pretty closely, so it might be a good source for us. As mentioned, though, our priority is to first define inclusion criteria for seas, straits, etc. Once we'll have done that, we'll see which notes need to be added to the deck and we'll work on sourcing or creating maps for them. Thanks for bumping the issue, I admit that I had forgotten about it. 😅 I'll try to get things moving asap.

**Comment by axelboc** [comment]
Alright, let's get the ball rolling!

### [physical-entities-v1.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5542659/physical-entities-v1.xlsx)

- I've created two sheets: one for **marginal seas** and one for **straits, channels and passages**.
- I've used Wikipedia's [List of seas](https://en.wikipedia.org/wiki/List_of_seas#Marginal_seas) as reference, which excludes lakes with "Seas" in their names, gyres, reefs, etc.
- I've listed seas that pass the following minimum requirements:
  - has an article on Wikipedia
  - has an article with an infobox that mentions the seas' area
  - has an area greater than 10,000 km2
  - is recognised by an official authority (i.e. this excludes notably the [Bermuda Triangle](https://en.wikipedia.org/wiki/Bermuda_Triangle))
- I've set the criterion for inclusion of marginal seas in the deck to **100,000 km2**. This gives us **29 seas to be added** to the deck and **1 to be removed**. Generally speaking, I quite like the result.
- I've listed every strait, channel and passage listed on Wikipedia, but I haven't picked inclusion criteria for them yet.

### Observations

- Requiring articles to have an infobox that mentions the seas' area made the sourcing process a lot easier. It also seems like a good indication of how "interesting" a sea is. However, it does exclude a few large seas, like [East China Sea](https://en.wikipedia.org/wiki/East_China_Sea), [Gulf of Carpentaria](https://en.wikipedia.org/wiki/Gulf_of_Carpentaria), [Sea of Japan](https://en.wikipedia.org/wiki/Sea_of_Japan), [Banda Sea](https://en.wikipedia.org/wiki/Banda_Sea), [Molucca Sea](https://en.wikipedia.org/wiki/Molucca_Sea), etc.) Rather than removing this minimum requirement, it may be wiser to edit the articles to improve their infoboxes.
- The [seas of the Arctic Ocean](https://en.wikipedia.org/wiki/List_of_seas#Arctic_Ocean) don't have infoboxes with areas and don't seem to be recognised by the IHO (whose 1953 document is still in effect). That being said, they're all very large and a [couple of them may be worth including](https://en.wikipedia.org/wiki/Weddell_Sea#/media/File:Antarctic-seas-en.svg) like [Weddell Sea](https://en.wikipedia.org/wiki/Weddell_Sea) or [Ross Sea](https://en.wikipedia.org/wiki/Ross_Sea).
- Some areas given by Wikipedia don't seem right -- those near the inclusion criterion may need to be reviewed:
  - the [Great Australian Bight](https://en.wikipedia.org/wiki/Great_Australian_Bight) as defined by the IHO seems much bigger than 46,000 km2;
  - the [Cook Inlet](https://en.wikipedia.org/wiki/Cook_Inlet) is said to have an area of 100,000 km2, which seems very approximative, since it looks much smaller than, say, the [White Say](https://en.wikipedia.org/wiki/White_Sea), which Wikipedia says has an area of 90,000 km2.
- There are quite a lot of seas in and around Indonesia that I fear may be hard to remember. The ones that have an area greater than 100,000 km2 are: [Arafura Sea](https://en.wikipedia.org/wiki/Arafura_Sea), [Banda Sea](https://en.wikipedia.org/wiki/Banda_Sea) (no area in infobox), [Celebes Sea](https://en.wikipedia.org/wiki/Celebes_Sea), [Flores Sea](https://en.wikipedia.org/wiki/Flores_Sea), [Java Sea](https://en.wikipedia.org/wiki/Java_Sea), [Molucca Sea](https://en.wikipedia.org/wiki/Molucca_Sea) (no area in infobox), [Sulu Sea](https://en.wikipedia.org/wiki/Sulu_Sea).
- Happy surprise, only few water bodies currently in the deck are absent from the list:
  - [Aral Sea](https://en.wikipedia.org/wiki/Aral_Sea), [Caspian Sea](https://en.wikipedia.org/wiki/Caspian_Sea), [Dead Sea](https://en.wikipedia.org/wiki/Dead_Sea), [Sea of Galilee](https://en.wikipedia.org/wiki/Sea_of_Galilee) (because they're lakes, not seas, so we'll ignore them for now);
  - [Gulf of Carpentaria](https://en.wikipedia.org/wiki/Gulf_of_Carpentaria), [Sea of Japan](https://en.wikipedia.org/wiki/Sea_of_Japan) (no area in Wikipedia infobox, which should be easy to correct).

**Comment by aplaice** [comment]
Wow, that's a huge amount of work!

I think it broadly makes sense. To the extent that I have doubts it's because of how arbitrarily named and ambiguously defined most water bodies are, and I'm not sure if any better choices are possible, given the data.

### Straits

> - I've listed every strait, channel and passage listed on the [List of seas](https://en.wikipedia.org/wiki/List_of_seas#Marginal_seas), but I haven't picked inclusion criteria for them yet.

Alternatively, we could use the (category) list of [International Straits](https://en.wikipedia.org/wiki/Category:International_straits), since it already includes the Straits of Gibraltar and the Bering Strait, and already constrains on one of the criteria that might be interesting (the strait being international (though that criterion definitely isn't *sufficient*)).  Slightly contrary to the name, it also includes "channels" such as the English Channel, the Mozambique Channel or the Drake Passage (under Antarctica), claiming that they're [actually straits](https://en.wikipedia.org/wiki/Category:English_Channel).  The list is far more extensive, though, and we probably aren't interested in most of the items...

As yet another alternative, a wikidata query that lists all the "straits" (according to Wikidata, the English Channel etc. are also straits) that "belong" to more than one country, could be used.  I'll try to write one (for comparison, even if we don't end up using it).

I'm a bit hesitant to just edit the List of Seas, to add in the Bering Strait and the Straits of Gibraltar, since it doesn't seem to be too reliable a source in this regard.  If it missed even them, it might also have missed many other straits, some of which it might make sense to add, but which we just wouldn't think of, ourselves.  OTOH we shouldn't let the perfect be the enemy of the good, so if none of the alternatives turns out unfeasible/impractical, just editing the List to add in the straits we want isn't *too* bad.

Regarding the inclusion criteria, I'm a bit stumped.  Arguably, the narrowness (and hence — usually — small area) of straits makes them *more* notable, since it increases their geopolitical importance.  I can't think of anything other than my (old) suggestion of including intercontinental straits, but that's also not ideal (e.g. would we add both [the Bosporus](https://en.wikipedia.org/wiki/Bosporus) and [the Dardanelles](https://en.wikipedia.org/wiki/Dardanelles)?).  The extensive straits (such as the English or Mozambique channels) could be included based on the same surface area criterion as all the other seas.


### Observations
>
> - Requiring articles to have an infobox that mentions the seas' area made the sourcing process a lot easier. It also seems like a good indication of how "interesting" a sea is. However, it does exclude a few large seas, like [East China Sea](https://en.wikipedia.org/wiki/East_China_Sea), [Gulf of Carpentaria](https://en.wikipedia.org/wiki/Gulf_of_Carpentaria), [Sea of Japan](https://en.wikipedia.org/wiki/Sea_of_Japan), [Banda Sea](https://en.wikipedia.org/wiki/Banda_Sea), [Molucca Sea](https://en.wikipedia.org/wiki/Molucca_Sea), etc.) 

> Rather than removing this minimum requirement, it may be wiser to edit the articles to improve their infoboxes.

Yes, that makes sense.  (Assuming that a reliable source is found (see below).  `mapshaper` allows obtaining the area of the features it plots, so I could get the area from our maps, but that goes against Wikipedia's "no original research"...)

> - The [seas of the Arctic Ocean](https://en.wikipedia.org/wiki/List_of_seas#Arctic_Ocean) don't have infoboxes with areas and don't seem to be recognised by the IHO (whose 1953 document is still in effect). That being said, they're all very large and a [couple of them may be worth including](https://en.wikipedia.org/wiki/Weddell_Sea#/media/File:Antarctic-seas-en.svg) like [Weddell Sea](https://en.wikipedia.org/wiki/Weddell_Sea) or [Ross Sea](https://en.wikipedia.org/wiki/Ross_Sea).

I assume you mean the seas of the Southern Ocean?  The very largest are probably worth including, since they really are quite huge, even if they're very far from any populated territories.

> - Some areas given by Wikipedia don't seem right -- those near the inclusion criterion may need to be reviewed:
>   - the [Great Australian Bight](https://en.wikipedia.org/wiki/Great_Australian_Bight) as defined by the IHO seems much bigger than 46,000 km2;

Yes, it seems that they're going by the Australian Hydrographic Service's definition.

>   - the [Cook Inlet](https://en.wikipedia.org/wiki/Cook_Inlet) is said to have an area of 100,000 km2, which seems very approximative, since it looks much smaller than, say, the [White Say](https://en.wikipedia.org/wiki/White_Sea), which Wikipedia says has an area of 90,000 km2.

Yes, Wikipedia's data is dubious.

> - There are quite a lot of seas in and around Indonesia that I fear may be hard to remember. The ones that have an area greater than 100,000 km2 are: [Arafura Sea](https://en.wikipedia.org/wiki/Arafura_Sea), [Banda Sea](https://en.wikipedia.org/wiki/Banda_Sea) (no area in infobox), [Celebes Sea](https://en.wikipedia.org/wiki/Celebes_Sea), [Flores Sea](https://en.wikipedia.org/wiki/Flores_Sea), [Java Sea](https://en.wikipedia.org/wiki/Java_Sea), [Molucca Sea](https://en.wikipedia.org/wiki/Molucca_Sea) (no area in infobox), [Sulu Sea](https://en.wikipedia.org/wiki/Sulu_Sea).

I wonder what Indonesians think of all the many Mediterranean seas. :D

On that note, the [Ionian](https://en.wikipedia.org/wiki/Ionian_Sea) and [Balearic Seas](https://en.wikipedia.org/wiki/Balearic_Sea) are also missing infoboxes. :)

### Additional comments

My main issue is that I have very serious doubts about Wikipedia's listed surface areas.  They don't seem to provide sources and they don't specify which (of the often many) definitions they're following.

OTOH assuming that they're approximately correct and that we verify the edge-cases, it probably doesn't matter.

**Comment by axelboc** [comment]
> I can't think of anything other than my (old) suggestion of including intercontinental straits, but that's also not ideal (e.g. would we add both the Bosporus and the Dardanelles?). The extensive straits (such as the English or Mozambique channels) could be included based on the same surface area criterion as all the other seas.

It's a good option. Another idea could be to include only straits that connect two marginal seas (and/or oceans) that both pass the inclusion criterion I suggested of 100,000 km2. This would exclude the Bosporus and the Dardanelles since they don't connect the Black Sea and the Mediterranean Sea directly.

Note that the English Channel doesn't pass the 100,000 km2 limit, which is why I didn't include it as a marginal sea. I think we'll need a different criterion for channels.

> Alternatively, we could use the (category) list of International Straits.

You're totally right, it includes a lot more straits, channels and passages. 💯 

> I'm a bit hesitant to just edit the List of Seas, to add in the Bering Strait and the Straits of Gibraltar, since it doesn't seem to be too reliable a source in this regard. If it missed even them, it might also have missed many other straits, some of which it might make sense to add, but which we just wouldn't think of, ourselves.

Yeah, it's not a good sign at all. It makes me wonder whether this List of Seas is exhaustive even for larger seas. I'd be interested in comparing it with the IHO's _[Limits of Oceans and Seas](https://epic.awi.de/id/eprint/29772/1/IHO1953a.pdf)_ of 1953, also to check whether the list (and our sublist) includes seas not delimited by the IHO.

> My main issue is that I have very serious doubts about Wikipedia's listed surface areas. They don't seem to provide sources and they don't specify which (of the often many) definitions they're following.

Yeah, the lack of sources is quite appalling. The fact that some of the numbers seem really approximative and inaccurate doesn't help, that's for sure. 😞 It also is a shame that quite a few large seas are missing area information altogether.

> `mapshaper` allows obtaining the area of the features it plots, so I could get the area from our maps, but that goes against Wikipedia's "no original research"...)

If our own "original research" is more exhaustive, precise and reliable than Wikipedia, I don't see a problem with it. As long as we're methodical about it and the research is well documented and fully reproducible, I don't think anybody would mind.

I've just found [a paper](https://www.semanticscholar.org/paper/A-New-Digital-Map-of-Limits-of-Oceans-and-Seas-with-Fourcy-Lorvelec/b692b0fe3cbb5278960502deb97f87f4782b76e4) introducing a digital map of the limits of oceans and seas based on the IHO's _[Limits of Oceans and Seas](https://epic.awi.de/id/eprint/29772/1/IHO1953a.pdf)_. Perhaps you had come across it before? I also found a site to [download the map's shapefile](https://www6.rennes.inrae.fr/ese_eng/HRMLOS), which we could totally pass through `mapshaper` to compute all the areas we need. What do you think?

**Comment by axelboc** [comment]
Or perhaps we may as well use the [2002 draft](http://wiki.geosys.ru/lib/exe/fetch.php/ru/portal/lib/iho/s23.los.ed4draft.2002.pdf) of the IHO's _Limits of Oceans and Seas_. After all, we already include the Southern Ocean, which does not exist in the 1953 version. 🤷‍♂️ I think we've talked about this before, but we could definitely highlight contentious areas between the 1953 and 2002 versions (like the limits of the East China Sea), a bit like we do for countries.

**Comment by aplaice** [comment]
> Yeah, the lack of sources is quite appalling. The fact that some of the numbers seem really approximative and inaccurate doesn't help, that's for sure. disappointed It also is a shame that quite a few large seas are missing area information altogether.

I've just spent some time looking at the Sea of Japan and the East China Sea, and part of the problem is that their infobox templates are not for water bodies, but for East Asian (or Chinese) items of interest.   (I think that this can be easily remedied, since (some) infoboxes can AFAICT be nested/embedded, though following [the guidelines](https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Infoboxes/embed) didn't seem to do anything, at least when previewing changes — it's possible that template changes don't apply when only previewing, that these particular infobox templates don't support embedding or that I did something wrong — I'll play around in a Wikipedia sandbox to check.)

The Sea of Japan and the East China Sea do both contain surface areas in Wikidata, but while adding/cross-referencing (respectively) the sources for them, it turned out that various sources provide areas differing from 978,000 km² to 1,048,950 km² (for the Sea of Japan) and from 750,000 km² to 1,249,000 km² (for the East China Sea).  Some of the sources even noted that multiple authorities provide different values...


> I've just found a paper introducing a digital map of the limits of oceans and seas based on the IHO's Limits of Oceans and Seas. Perhaps you had come across it before? I also found a site to download the map's shapefile, which we could totally pass through mapshaper to compute all the areas we need. What do you think?

That looks great!  I *might* have seen it before and rejected it due to the non-commercial license for the shapefiles (which would make the SVG derived from the data incompatible with the CC BY-SA or similar needed for Wikimedia), though I don't remember for sure.  In any case, for calculating the areas it's perfect.  I'll definitely have a look!  (I'm not making any promises on when I'll do everything that I've promised to do, in this thread, though I'll try soon-ish:))

> It's a good option. Another idea could be to include only straits that connect two marginal seas (and/or oceans) that both pass the inclusion criterion I suggested of 100,000 km2. This would exclude the Bosporus and the Dardanelles since they don't connect the Black Sea and the Mediterranean Sea directly.

That's a great and simple solution!

> Note that the English Channel doesn't pass the 100,000 km2 limit, which is why I didn't include it as a marginal sea. I think we'll need a different criterion for channels.

I hadn't realised. :O  Your suggested criterion of connecting two sufficiently large seas would work in this case, though — both the Celtic Sea and the North Sea have an area greater than 100,000 km².

<hr/>

> Or perhaps we may as well use the 2002 draft of the IHO's Limits of Oceans and Seas. After all, we already include the Southern Ocean, which does not exist in the 1953 version. man_shrugging I think we've talked about this before, but we could definitely highlight contentious areas between the 1953 and 2002 versions (like the limits of the East China Sea), a bit like we do for countries.

It's a really neat idea and it'd look great, but I'm not sure whether there are sufficiently good shapefiles for the 2002 version, though.  (I had manually input the data from the 2002 draft for the Bering Strait map, with QGIS, and it took a while.  Experience would speed things up considerably, but I still wouldn't relish doing it for all of the seas.)  There aren't really sufficiently good, appropriately licensed shapefiles even for the published 1953 version — I had to patch the mostly great Natural Earth data with alternative sources, in a couple of cases, and the Baltic Sea is still subtly wrong (it's been on my to-do-list for a while).

**Comment by axelboc** [comment]
The good news is that a lot of the boundaries [didn't change between the 1953 and 2002 versions](https://unstats.un.org/unsd/geoinfo/ungegn/docs/Training/Manila/3_Naming_objects_beyond_a_single_sovereignty.pdf), so I think the 1953 shapefile would cover a good 80% of our need.

**Comment by axelboc** [comment]
> That looks great! I might have seen it before and rejected it due to the non-commercial license for the shapefiles (which would make the SVG derived from the data incompatible with the CC BY-SA or similar needed for Wikimedia), though I don't remember for sure. In any case, for calculating the areas it's perfect.

From what I gather, the shapefile is available for non-commercial use, so we could definitely reference it in this repo. 👍

**Comment by axelboc** [comment]
Since Wikipedia's _List of seas_ is so unreliable, here is the list of all the seas (including straits) described in the IHO's 2002 draft:

**[IHO seas.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5547398/IHO.seas.xlsx)**

The draft often uses the seas' local names, so I've normalized them all to English based on Wikipedia. Only very few seas don't have articles on Wikipedia (that I could find): Aru Sea (relatively large sea off the coast of Papua New Guinea), Central Baltic Sea (portion of the Baltic Sea), Sound Sea (tiny sea off the coast of Estonia), Tryoshnikova Gulf (tiny gulf off the coast of Antarctica). Also, Wikipedia has an article for the Northwest Passage (i.e. the sea route in Northern Canada), but not for what the IHO calls the Northwestern Passages (i.e. the combination of all the waterways in the region).

Regardless, I think this is a much better list than what I had before. It looks a lot more complete, especially when it comes to straits. If you agree, @aplaice, I'll update `physical-entities` with this list, separating straits from seas, and we can go from here.

I've crossed checked the list with the IHO's 1953 document and identified the seas that appear in both versions. The next step will be for me to check which seas have changed boundaries between the 1953 and 2002 versions. This will inform us on which areas need to be calculated from scratch, and which can be extracted from the shapefile I mentioned previously.

Note that the 1953 version includes one sea that is absent from the 2002 version: the Sea of Japan. This is due to the naming dispute between Japan and Korea. I think it should still be included in the list, though.

Note also that a number of seas from Wikipedia's _List of seas_ are absent from the IHO's document. The most significant ones are the Cook Inlet (the area of which Wikipedia most likely overestimates), the Argentine Sea (which lacks international recognition), and the Levantine Sea (which some sources characterise as a lake). I don't think any of these are worth keeping in `physical-entities`.

**Comment by aplaice** [comment]
> Regardless, I think this is a much better list than what I had before. It looks a lot more complete, especially when it comes to straits. If you agree, @aplaice, I'll update physical-entities with this list, separating straits from seas, and we can go from here.

I think that it makes sense to replace Wikipedia's *List of seas* with the 2002 IHO draft.  I'm far less confident about replacing the previous criterion of "existence of inbobox" + "area in infobox > 100,000 km²", with the criterion of our own calculated (based on the 2002 draft) area being greater than 100,000 km² or a wholescale editing of Wikipedia's infoboxes.

Firstly, updating the shapefile will take a considerable amount of work (to do semi-properly), (assuming that the [comparison document](https://unstats.un.org/unsd/geoinfo/ungegn/docs/Training/Manila/3_Naming_objects_beyond_a_single_sovereignty.pdf#page=16) is correct, I'd assume approx. half a day) for something that will be of limited general use — due to the non-commercial license of the original source it won't be useful for anything Wikipedia-related, and since it'll be done only semi-properly, it won't be of much use in an academic environment. (Doing it actually *properly* in the way the source 1953-based article did would be an immense undertaking.) (Though an automated parsing of the text of the IHO 2002 draft might be feasible (as you've probably noticed, I like automated solutions, since they don't leave me second-guessing whether I didn't make a mistake :).)

Secondly, the 2002 draft, due to its very nature, isn't really an authoritative, indisputable source, so I'd be uncomfortable editing Wikipedia to replace/provide an area calculated based on it.  (In cases like the extent of the South China Sea, it even gets a bit political...)

Thirdly, for purely our purposes (since it wouldn't be easily/cleanly "upstreamable"), I feel like it's a huge overkill. For the edge cases, it makes sense to calculate the areas, to double-check Wikipedia's dubious values, but otherwise even large errors in the values are unlikely to change whether a sea passes the criterion.

I feel really bad discouraging/opposing your enthusiasm. :(

> the Sea of Japan [...] I think it should still be included in the list, though.

Yes, definitely.

> Note also that a number of seas from Wikipedia's List of seas are absent from the IHO's document. I don't think any of these are worth keeping in physical-entities.

Yes, I think that it's safe to exclude them.

**Comment by axelboc** [comment]
Oh, sorry, I wasn't suggesting creating a full-on shapefile with the 2002 data. Here is the process I have in mind:

1. List the seas defined by the IHO in 1953 and 2002, and use the 2002 seas as our source list.
1. Get the areas of the 1953 seas from the shapefile.
1. Do a rough visual comparison of the outlines of the seas in the 1953 shapefile with the outlines of the seas on the maps of the 2002 draft. Indicate whether each sea's area has increased/decreased significantly in proportion to its size.
1. Measure _very roughly_ in QGIS all the seas that were added in the 2002 draft.
1. Each sea should now have an area, whether precise or not. Apply the threshold of 100,000 km2 to the list and confirm that it's the one we want to use.
1. Identify the seas that have, or may have, an area close to the 100,000 km2 threshold, taking into consideration the observations from step 3. Measure each of these seas more accurately on QGIS.
1. For areas that remain very close to the threshold, compare them with areas provided on Wikipedia.
1. Review the final inclusion list and discuss any contentious cases.

I'm currently half-way through step 3. Do you think the process makes sense? Am I going in the right direction?

**Comment by aplaice** [comment]
Yes, the process makes sense! Sorry for the confusion!

Regarding 2: it turns out to be even easier than expected. The areas can be trivially calculated and neatly outputted with

```bash
mapshaper  HRmLOS_1.1.shp -each 'area=this.originalArea' -o areas.csv
```

but it turns out that even this is not needed, since in this particular case, the shapefiles already contained the relevant areas (though for some reason, all the areas are out by a factor of 1.002258 compared to those calculated by `mapshaper` — I'm not yet sure which is correct, but I'll investigate by working with some other shapefiles with known areas etc.).

All that remains is consolidating the areas in the cases where a sea corresponds to more than one unit (e.g. the Mediterranean or the Baltic). I'll upload the data tomorrow.

> Measure very roughly in QGIS all the seas that were added in the 2002 draft.

As a second (additional) sanity check, the areas of the seas as obtained from Natural Earth, could also be calculated. (In some cases, NE had subdivisions of seas that were not in the 1953 version.)

**Comment by axelboc** [comment]
> Regarding 2: it turns out to be even easier than expected. The areas can be trivially calculated and neatly outputted with

Indeed, I created a calculated field with the area (rounded to the nearest integer) in QGIS, so I already have all the 1953 areas. I've also combined the areas of the Mediterranean and Baltic seas based on their 1953 boundaries (which are the same, overall, as the 2002 boundaries).

I'm now half-way into comparing the boundaries visually between the shapefile and the 2002 draft (which has maps with outlines for every sea, unlike the 1953 document... 😄 which should make creating the maps for the included seas much easier, by the way).

**Comment by aplaice** [comment]
> Indeed, I created a calculated field with the area (rounded to the nearest integer) in QGIS, so I already have all the 1953 areas. I've also combined the areas of the Mediterranean and Baltic seas based on their 1953 boundaries (which are the same, overall, as the 2002 boundaries).

Great! :) 

> which should make creating the maps for the included seas much easier, by the way).

Yeah, I had used the maps from the 2002 draft (as [multipe `.doc`s ](https://web.archive.org/web/20160301222056/http://www.iho.int/mtg_docs/com_wg/S-23WG/S-23WG_Misc/Draft_2002/Draft_2002.htm), from the IHO website — the single PDF you found, is much nicer), as an additional, if partial, sanity-check when making the maps.

> As yet another alternative, a wikidata query that lists all the "straits" (according to Wikidata, the English Channel etc. are also straits) that "belong" to more than one country, could be used. I'll try to write one (for comparison, even if we don't end up using it).

Now, done [here](https://query.wikidata.org/#SELECT%20DISTINCT%20%3Fstrait%20%3FstraitLabel%20%3FnoCountries%20%3Farticle%20WHERE%20%7B%0A%20%20%7B%0A%20%20%20%20SELECT%20%3Fstrait%20%3Farticle%20%28COUNT%28DISTINCT%20%3Fcountry%29%20AS%20%3FnoCountries%29%20WHERE%20%7B%0A%20%20%20%20%20%20%3Fstrait%20wdt%3AP31%20wd%3AQ37901%3B%0A%20%20%20%20%20%20%20%20wdt%3AP17%20%3Fcountry.%0A%20%20%20%20%20%20%3Farticle%20schema%3Aabout%20%3Fstrait%3B%0A%20%20%20%20%20%20%20%20schema%3AisPartOf%20%3Chttps%3A%2F%2Fen.wikipedia.org%2F%3E.%0A%20%20%20%20%7D%0A%20%20%20%20GROUP%20BY%20%3Fstrait%20%3Farticle%0A%20%20%7D%0A%20%20FILTER%28%3FnoCountries%20%3E%201%20%29%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22en%22.%20%7D%0A%7D%0AORDER%20BY%20%28%3FstraitLabel%29).

<details><summary>Query also here, in case GitHub mangles the long link</summary>

```sql
SELECT DISTINCT ?strait ?straitLabel ?noCountries ?article WHERE {
  {
    SELECT ?strait ?article (COUNT(DISTINCT ?country) AS ?noCountries) WHERE {
      ?strait wdt:P31 wd:Q37901;
        wdt:P17 ?country.
      ?article schema:about ?strait;
        schema:isPartOf <https://en.wikipedia.org/>.
    }
    GROUP BY ?strait ?article
  }
  FILTER(?noCountries > 1 )
  SERVICE wikibase:label { bd:serviceParam wikibase:language "en". }
}
ORDER BY (?straitLabel)
```

</details>

[straits_wikidata.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5556382/straits_wikidata.xlsx)

I still can't think of any sensible criteria to narrow down the list from the 67 straits, though.

**Comment by aplaice** [comment]
I've applied the criterion of connecting two water bodies each having an area > 100,000 km² to the list of "International" Straits (those bordering more than one country), from Wikidata, and it results in 18 matches:

[straits_wikidata.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5562670/straits_wikidata.xlsx)

The areas are taken from your `physical-entities-v1.xlsx`, so a couple sea areas are still missing (due to lack of Wikipedia infobox), and hence some more straits might fit the criteria. (I gave the oceans (and parts of oceans) an arbitrary, large area of 999999 km², since they obviously pass.)

All of the matching straits are "interesting", at least for me, but I feel that there are too many of them...

Slightly arbitrarily, I'd cut down on some of the many straits between the Caribbean Sea and Atlantic Ocean, replace the [Beagle Channel](https://en.wikipedia.org/wiki/Beagle_Channel) with the [Drake Passage](https://en.wikipedia.org/wiki/Drake_Passage) or the [Straits of Magellan](https://en.wikipedia.org/wiki/Strait_of_Magellan) and remove the [Strait of Bonifacio](https://en.wikipedia.org/wiki/Strait_of_Bonifacio), as well as perhaps the [Balabac Strait](https://en.wikipedia.org/wiki/Balabac_Strait).

**Comment by axelboc** [comment]
Awesome! Yeah, it feels like a lot, but maybe it's because the list contains some obscure entries. I feel like 12~15 would be a good target range.

Looking at your list and at the missing straits, I wonder whether the "more than one country" rule isn't a bit too arbitrary. Also, I'm concerned as to the recognition status and international relevance of some of the straits and channels returned by Wikidata.

What do you think about the straits, channels and passages listed in the IHO draft of 2002? I count 24 of them (listed below). I feel like it might be a more relevant starting list. That being said, it doesn't include the Strait of Magellan, which I had completely forgotten about. It's probably because it's not in international waters. 😞 

- Bass Strait
- Bering Strait
- Bransfield Strait
- Danish straits (The Sound, Little Belt, Great Belt)
- Davis Strait
- Drake Passage
- English Channel
- Hudson Strait
- Makassar Strait
- Mozambique Channel
- Northwestern Passages
- Palk Strait
- Singapore Strait
- Strait of Dover
- Strait of Gibraltar
- Strait of Hormuz
- Strait of Malacca
- Strait of Sicily
- Strait of Tartary
- Straits of Florida
- Sumba Strait
- Sunda Strait
- Taiwan Strait
- Torres Strait

What I like about this list is that:

- it has an international authority behind it (and I don't think the fact that it's a draft diminishes this aspect in any way);
- it's much smaller to start with, so we might be able to filter it down more easily;
- it's small-enough that we could append another list to it (for straits that lie in national waters, if that's indeed the reason why the Strait of Magellan is not already included);
- a good number of the straits it contains are in your list and most of those pass the "connecting seas" criterion.

---

Back to the problem of the criteria, One significant problem with the "connecting seas" criterion, as you've noticed, is granularity. In many cases, a strait may connect a sea that is part of a larger sea. How do we decide which of the two seas to take into account? 

Perhaps we should start again with your initial idea of including straits that connect two oceans (or was it two continents? or either?) How many straits in your list and out of the 24 defined by the IHO would this apply to?

In parallel, we could apply a simple area criteria in order to include the biggest channels and passages, like the Northwestern Passages, the Drake Passage, the Mozambique Channel and the English Channel.

A few interesting straits would probably still be left out. Assuming the data exists, we might be able to pick, out of the remaining straits, the 5 or so that get the most maritime traffic ... or something in this vein. 🤷‍♂️ 

--- 

Here is where I'm at, for info: **[IHO seas.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5563021/IHO.seas.xlsx)**. I've collated the areas of all the 1953 seas, compared them visually with the 2002 maps, and identified whether they had significantly/insignificantly increased/decreased.

**Comment by axelboc** [comment]
Ooh I just learnt about the maritime law concept of [Transit passage](https://en.wikipedia.org/wiki/Transit_passage). The article lists 5 straits as being covered by the transit passage provisions: the Strait of Gibraltar, Dover Strait, Strait of Hormuz, Bab-el-Mandeb and Strait of Malacca. It also mentions that the Danish Straits, the Turkish Straits and the Strait of Magellan are not covered by the provisions because they are already governed by "international conventions".

Maybe we could use this article to complement the IHO list, which would amount to adding the Turkish Straits, Bab-el-Mandeb and the Strait of Magellan?

**Comment by aplaice** [comment]
> Looking at your list and at the missing straits, I wonder whether the "more than one country" rule isn't a bit too arbitrary.

Yeah, it is a bit arbitrary.

> What do you think about the straits, channels and passages listed in the IHO draft of 2002? I count 24 of them (listed below). I feel like it might be a more relevant starting list.

Yeah, I think you might be right!

<hr/>

I've now added the information about whether the strait joins two oceans or separates two continents (or continental plates), for both the Wikidata straits and the IHO+transit passage ones.

However, the boundary between the Pacific and Indian Oceans is pretty much undefined, since the 2002 draft doesn't state to which the East China and Archipelagic Seas belong. The "best" that I could find was the borders of the oceans from the [CIA factbook maps](https://en.wikipedia.org/wiki/Borders_of_the_oceans) used on Wikipedia...

[straits_wikidata.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5563435/straits_wikidata.xlsx)

I think that `OR(connects oceans, separates continents, is transit passage)` for the IHO+transit passage straits gives a relatively sensible result, and a relatively manageable number of 14. It excludes Denmark Strait, but AFAICT it's not really that important, interesting tidbit that it's between Europe and America, aside. (However, the fact that it adds the "Danish straits" instead, might confuse some people!) The other logical combinations would also work, though.

I'm slightly worried that we're getting inured to straits, and even 14 (or 10) will be too much...

<hr/>

> Here is where I'm at, for info: IHO seas.xlsx. I've collated the areas of all the 1953 seas, compared them visually with the 2002 maps, and identified whether they had significantly/insignificantly increased/decreased.

Wow! It looks great!

> Ooh I just learnt about the maritime law concept of Transit passage.

That's pretty cool!

**Comment by axelboc** [comment]
I've tried to find what the busiest straits might be, but I couldn't find a simple list. All I could find were [maps of shipping routes](https://www.vox.com/2016/4/25/11503152/shipping-routes-map). The busiest straits are easily identifiable: Taiwan, Malacca, Bab-el-Mandeb, Sicilly, Bosphorus/Turkish, Gibraltar, English Channel, Dover, Danish. This is far from ideal, though:

- it's very approximative, of course;
- it's difficult to tell whether some of the traffic actually crosses the strait or arrives at a port located near the strait;
- it doesn't consider the length of the shipping routes (i.e. long, international shipping routes are probably more relevant);
- it doesn't include all maritime traffic (cruises, fishing, etc.)

I've also looked at [oil choke points](https://www.eia.gov/international/analysis/special-topics/World_Oil_Transit_Chokepoints), which seems interesting for a geo-political standpoint: Hormuz, Malacca, Bab-el-Mandeb, Danish, Turkish. They're all included in the transit passage list, but the latter also has Magellan, Dover and Gibraltar.

---

The combination I like the most so far is `OR(connects oceans, is transit passage)`, which gives 13 straits (or 14 with the Sunda Strait). I think it's good that the Denmark Strait is not included. That being said:

- If I had to pick one, I would prefer to have the English Channel over the Strait of Dover.
- I don't feel strongly about Bass Strait and Drake Passage being included at all... maybe even the Northwestern Passages, but I couldn't find a way to remove them other than to remove the "connecting oceans" criterion, which would also remove Bering and Davis. Bering passes the continents criterion, but this criterion also includes the Strait of Sicily, which I don't find very interesting.
- I'm not sure how I feel about the Danish and Turkish straits. On one hand they don't seem like very interesting notes to include; on the other hand they wouldn't get too much in the way and we would probably mention the actual straits they contain in the _Country info_ field. We could of course arbitrarily exclude them on the basis that they are not a single strait; we could also replace the Turkish Straits with the Bosphorus Strait on the basis that it is more relevant geo-politically (cf. choke points).
- If we wanted to bring the number of 13 or 14 down a little more, we could use the transit passage as a criterion but not as a source, leaving only the IHO straits. This would remove the Turkish Straits, Bab-el-Mandeb and the Strait of Magellan.

A lot of very subjective choices and feelings in all of this... but we're still brainstorming 😅

**Comment by aplaice** [comment]
> I've tried to find what the busiest straits might be, but I couldn't find a simple list. [...]

Yeah, it's quite interesting, butI don't see a straightforward way of extracting something useful from this.

<hr/>

> If I had to pick one, I would prefer to have the English Channel over the Strait of Dover.

Yes, definitely.

> I don't feel strongly about Bass Strait and Drake Passage being included at all... maybe even the Northwestern Passages,

Yeah.

> the Strait of Sicily, which I don't find very interesting.

TBH I'd also prefer if it weren't included.

> I'm not sure how I feel about the Danish and Turkish straits.

I'd lean towards preferring them excluded. Their constituent parts could definitely be included, in the *Country info*, but that would, in effect, make the relevant cards *about* the *Country info*, since the "main answer" is rather uninteresting.

The exclusion of "multiple" straits would also have the mild benefit of excluding the Northwestern Passages.

<hr/>

I should probably take a brief break from this, since I've now added three more fields (is in IHO 2002, is "international" (bordering more than one country) and is a single strait), in the interest of finding a combination that would justify my preferences, which is bordering on the slightly crazy...

[straits_wikidata.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5576524/straits_wikidata.xlsx)

If I were to make a decision now, I'd probably vote for `((connects oceans) OR (is transit passage)) AND (single strait)`, but I'm not quite happy about it.

**Comment by axelboc** [comment]
### [IHO seas.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5579526/IHO.seas.xlsx)

I've finished measuring the 2002 areas that needed to be measured. I used the area measuring tool in QGIS, along with a variety of equal-area projections. I think the result is sufficiently precise for our purpose.

I've experimented with various area thresholds, and the results give quite high inclusion numbers. I've tried adding a second criterion to remove most of the Antarctic seas, which are quite numerous and, in my opinion, not very interesting. (I also had trouble measuring some of them because of significant changes in the coastline of Antarctica since 1953.)

Here are the results of my experiments:

| Criteria                                           | Pass | In deck | Add | Remove |
|----------------------------------------------------|------|---------|-----|--------|
| >= 100,000 km2                                     | 77   | 26      | 51  | 1      |
| >= 150,000 km2 | 69   | 25      | 44  | 2      |
| >= 100,000 km2 OR >= 500,000 km2 if Antarctic     | 68   | 26      | 42  | 1      |
| **>= 175,000 km2** | 66   | 25      | 41  | 2      |
| >= 200,000 km2 | 62   | 22      | 40  | 5      |
| **>= 150,000 km2 OR >= 500,000 km2 if Antarctic** | 61   | 25      | 36  | 2      |
| **>= 175,000 km2 OR >= 500,000 km2 if Antarctic** | 60   | 25      | 35  | 2      |
| >= 200,000 km2 OR >= 500,000 km2 if Antarctic     | 56   | 22      | 34  | 5      |

In my opinion, the 100,000 km2 threshold is way too inclusive, and the 200,000 km2 threshold removes too many seas from the deck (2 that get excluded in most cases: White Sea and Adriatic Sea; plus 3 more: Aegean Sea, Gulf of California, Bay of Biscay).

I've highlighted the criteria that I think are the best. My preference goes to **>= 175,000 km2 OR >= 500,000 km2 if Antarctic**.

**Comment by aplaice** [comment]
This is amazing (as expected :))!

Two things that slightly bother me, but I don't see any great solutions:

1. That's quite a large number of new seas/notes (an over 100% increase).

2. Some seas that are being excluded, (subjectively) feel more interesting than some that are being included. For instance, I feel that the Adriatic Sea is slightly more note-worthy than the Ionian or the Tyrrhenian Sea, and that the White Sea is slightly more interesting than, say, the Laptev Sea.

<hr/>

Perhaps we could introduce the same higher threshold for the subdivisions of the Arctic Ocean that you suggested for the Antartic (Southern Ocean) seas? At `500,000 km²` it would AFAICT exclude [Chukchi Sea](https://en.wikipedia.org/wiki/Chukchi_Sea) and [Iceland Sea](https://en.wikipedia.org/wiki/Iceland_Sea).

I'd consider lowering the non-polar seas threshold to 125,000.  Compared to 175,000 it'd include the Adriatic Sea (which I'd miss), the [Gulf of Tonkin](https://en.wikipedia.org/wiki/Gulf_of_Tonkin) (which is [vaguely historically important](https://en.wikipedia.org/wiki/Gulf_of_Tonkin_incident)) and the [Seram Sea](https://en.wikipedia.org/wiki/Seram_Sea) (which I don't have a strong opinion about, either way).  However, I'm not sure if it doesn't bloat the deck too much, so 175,000 might indeed be better (it's definitely better than 100,000, 150,000 or 200,000).

**Comment by axelboc** [comment]
Here is another idea: instead of applying the 500,000 km2 threshold to polar seas, we apply it to seas introduced in the IHO's 2002 draft. Seas that were already defined in 1953 can then be applied the lower threshold of 125,000 km2.

- It allows the Adriatic Sea and the Seram Sea to be included, but not the Gulf of Tolkin.
- It doesn't change anything for the Southern Ocean seas, which were all added in 2002.
- It removes the Natuna Sea, the Aru Sea and the Iceland Sea, none of which seem crucial to keep.
- It brings the number of seas to be added to the deck down a bit more, to 33, for an inclusion total of 59.
- It seems to make good sense: recently added seas are more interesting because of their size rather than their history.

![image](https://user-images.githubusercontent.com/2936402/99910121-9b0a9c80-2cec-11eb-94c1-9c851a4e6a28.png)

**Comment by aplaice** [comment]
That seems more-or-less perfect!

**Comment by axelboc** [comment]
### [physical-entities.xlsx](https://github.com/axelboc/anki-ultimate-geography/files/5860690/physical-entities.xlsx)

I've had another look at the straits and I may have found a decent set of criteria: 

**`is transit OR (area <= 50,000 AND >= 2 states)`**

The source list I've used combines the straits of the IHO draft of 2002 with the transit passage straits, but excludes the "collective straits" (_Turkish straits_ and _Danish straits_). Here is the result:

![image](https://user-images.githubusercontent.com/2936402/105610074-c4f44400-5dad-11eb-8aa7-6ff43a6e591d.png)

![image](https://user-images.githubusercontent.com/2936402/105610059-98d8c300-5dad-11eb-92ce-39037c8eefc7.png)

The "to be removed" count includes _Denmark Strait_ even though it's not part of the source list. What do you think?

**Comment by aplaice** [comment]
That looks reasonable!

It'd be a shame to exclude the English Channel, while including the Mozambique channel doesn't feel like an issue, so I'd vote for the separate "channel" criterion.
